test/integration/services/modal.js

```diff
@@ -0,0 +1,144 @@
+// import _ from 'underscore';
+
+context('Modal Service', function() {
+  const smallOnTall = () => {
+    cy
+      .getRadio(Radio => {
+        Radio.request('modal', 'show:tall', {
+          headingText: 'Tall Modal Header',
+          bodyText: 'Tall Modal, which is under the small modal.',
+        });
+      });
+
+    cy
+      .getRadio(Radio => {
+        Radio.request('modal', 'show:small', {
+          headingText: 'Small Modal Header',
+          bodyText: 'Small Modal, this guy shows over the top of the tall modal.',
+        });
+      });
+  };
+  specify('Displaying', function() {
+    cy
+      .visit('/');
+
+    smallOnTall();
+
+    cy
+      .get('.modal--small')
+      .within(() => {
```

Comment:
another way this test could be written to get around 'contains' changing the context would be:
```js
cy
 .get('.modal--small')
 .should('contain', 'Small Modal')
 .find('.modal-cancel')
 .click();
```
I'm not a huge fan of `within` unless it's providing a significant reduction in complexity.